Animations/Hardik-Ghori.md

```diff
@@ -0,0 +1,4 @@
+name: Hardik Ghori
```

Comment:
Hey @Hardik-Ghori 👋, Just one change!
this md file should be in your username folder only!

Reply:
hello jay,
Is it good now?